lib/components/map/enhanced-stop-marker.tsx

```diff
@@ -166,12 +163,11 @@ class EnhancedStopMarker extends Component<Props> {
   }
 }
 
-const mapStateToProps = (state: any, ownProps: OwnProps) => {
+const mapStateToProps = (state: AppReduxState, ownProps: OwnProps) => {
   const { highlightedStop } = state.otp.ui
 
-  const transitModes = state.otp.config.modes.transitModes
   const modeColors: ModeColors = {}
-  transitModes.forEach((mode: ConfiguredTransitMode) => {
+  state.otp.config.modes.transitModes.forEach((mode) => {
```

Comment:
Do you want to also add the
`const { config: otpConfig } = state.otp` here?

Reply:
Not needed, in this file, it is used only on this line.